### DISCUSSION TITLE
doris从1.2.3 升级到doris-2.0.0-rc04-96bb9a7,原来使用的java udf,再使用时，会直接导致be 宕掉问题

### DISCUSSION THREAD

**Discussion by jhlon** [apache/doris#23670] [A - General / Q&A]
#客户端查询时，be进程会直接挂掉。 ase_show（string）是udf
select 
nhlh_hcm.ase_show(drift) 浮动薪酬
       ,nhlh_hcm.ase_show(other)      其他薪资
       ,salary_calculate_type   算薪类型
           ,lpad((ROW_NUMBER() OVER (
           ORDER BY l1_deptid_id desc
                   ,l2_deptid_id desc
                   ,l3_deptid_id desc
                   ,l4_deptid_id desc
                   ,l5_deptid_id desc
                   ,l6_deptid_id desc
                   ,l7_deptid_id desc
                   , salary_group_id desc
                   , emp_id asc
         )),7,'0') as row_num
from   dwd_emp_salary_item2;
ERROR 1105 (HY000): RpcException, msg: org.apache.doris.rpc.RpcException: io.grpc.StatusRuntimeException: UNAVAILABLE: Network closed for unknown reason。

**Comment by LemonLiTree** [comment]
Take a look at the code of udf
